test/samlp.tests.js

```diff
@@ -96,6 +96,20 @@ describe('samlp (unit tests)', function () {
       });
     });
 
+    it('should return a decrypted assertion when using tripledes-cbc', function (done) {
```

Comment:
This is one of the existing algorithms that is now considered "insecure". This test is mainly to check that there wasn't a breaking change.